CSS/docs.css

```diff
@@ -0,0 +1,77 @@
+@import url(/CSS/main.css);
+@import url(/CSS/nav.css);
+
+.doc-container{
+    margin-top:2.25rem;
+    display:grid;
+    grid-template-columns: 20% auto;
+ 
+}
+
+.sidebar-section{
+    font-size:1.25rem;
+}
+
+.main-content{
+
+    width:98%;
+    margin:1rem 0 0 auto ;
+    height:inherit;
+    font-size: 2rem;
+}
+
+.main-content-display{
+    display: flex;
+    background-color: var(--background);
+    border-radius: 4px;
+    background-color: #F5F7F9;
```

Comment:
you may use this too as a color variable

Reply:
okay